doc/kj.md

```diff
@@ -143,7 +141,7 @@ you initialize it otherwise:
 class Continent {
   ...
 
-  Continent(Thing creature) : creature(creature) {};
+  Continent(kj::Own<Thing> creature) : creature(creature) {};
```

Comment:
Haven't tested, but I suspect the initializing value needs to be `kj::mv()`ed into the member for the constructor to compile without errors.  Also, extra semicolon.

Reply:
You would absolutely be right and good catch. Fixing.